lib/utils.js

```diff
@@ -392,27 +412,46 @@ function mergeSettings(target, src) {
       }
     });
   } else {
+    let baseModelAclRank;
     if (target != null && typeof target === 'object') {
       // Add properties from target to dst
       Object.keys(target).forEach(function(key) {
+        if (key === 'acls' && Array.isArray(target[key])) {
+          let targetACLs = target[key];
+          targetACLs.forEach(function(acl) {
+            acl.rank = acl.rank || 1;
+            // capture the base model ACL rank
+            baseModelAclRank = baseModelAclRank || acl.rank;
```

Comment:
as new acls are pushed in the array, the last one always has the highest rank so there's no need to check for the max.
but i'll go on with you proposal to make it more solid.